script/release.js

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env node
+
+const { versionBump, currentVersion }  = require('./version-utils');
+const { logger, spawnOrFail, prompt, shouldContinuePrompt, quit, process, path } = require('./cli-utils');
+
+const deployDemo = (version) => {
+  const demo_name = `chime-sdk-demo-${version.replace(/\./g, "-")}`;
+  logger.log(`Deploying ${demo_name} ...`);
+  process.chdir(path.join(__dirname, '../demos/serverless'));
+  spawnOrFail('npm', [`run deploy -- -b ${demo_name} -s ${demo_name} -o ${demo_name} -u false`], { printErr: true });
+};
+
+const getCurrentRemoteBranch = () => {
+  return (spawnOrFail('git', ['for-each-ref --format="%(upstream:short)" "$(git symbolic-ref -q HEAD)"'], { skipOutput: true })).trim();
+};
+
+const buildAndPack = () => {
+  logger.log('Building package...');
+  spawnOrFail('npm', ['run build:release']);
+  logger.log('Packaging ...');
+  spawnOrFail('npm', ['pack --dry-run'], { printErr: true });
+};
+
+const cleanUp = async (remoteBranch) => {
+  logger.warn(`Warning: Resetting HEAD ${remoteBranch ? `to ${remoteBranch}` : ''}.\nAll current staged and local changes will be lost.`);
+  await shouldContinuePrompt();
+  spawnOrFail('git', [`reset --hard ${remoteBranch ? remoteBranch : ''}`]);
+  spawnOrFail('git', [' clean -ffxd .']);
+};
+
+const release = async () => {
+  spawnOrFail('git', ['fetch origin'], { skipOutput: true });
+  const currentBranch = (spawnOrFail('git', [' branch --show-current'], { skipOutput: true })).trim();
+  const remoteBranch = getCurrentRemoteBranch();
+  if (!remoteBranch || (remoteBranch !== 'origin/main' && (/^origin\/release-[0-9]+\.x$/).test(remoteBranch))) {
+    logger.error(`The local branch ${currentBranch} does not track either main or release-<version>.x branch`);
+    quit(1);
+  }
+  await cleanUp(remoteBranch);
+
+  buildAndPack();
+  logger.log('Do you want to upload these files to release branch?\n');
+  await shouldContinuePrompt();
+  spawnOrFail('git', ['push origin HEAD:release -f']);
+  deployDemo(currentVersion);
+
+  //Bump next development version
+  await versionBump();
+};
+
+const hotfix = async () => {
+  await cleanUp();
+
```

Comment:
Do not want to raise any bar on the release script but would be good to remove these extra spaces :) or follow consistency across release, hotfix and relevant functions which use these calls.

Reply:
What extra space? I prefer more empty lines in the code.

Reply:
I mean if we want new lines then we can use it more consistently in other functions as well. Like for example in release function no new line after buildAndPack, but here it does have it so not sure what criteria for this but up to you if to preserve reading consistency. Something minor.

Reply:
Addressed :)